src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figure = new Figure[6];
```

Comment:
Instead of directly initializing the array with a fixed size, consider defining a constant for the number of figures to improve code readability and maintainability.